sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -191,6 +192,25 @@ public static SentryEnvelopeItem fromCheckIn(
     return new SentryEnvelopeItem(itemHeader, () -> cachedItem.getBytes());
   }
 
+  public static SentryEnvelopeItem fromMetrics(final @NotNull EncodedMetrics metrics) {
+
+    final CachedItem cachedItem =
+        new CachedItem(
+            () -> {
+              // avoid method refs on Android due to some issues with older AGP setups
+              //noinspection Convert2MethodRef
+              return metrics.getStatsd();
+            });
+
+    final @NotNull SentryEnvelopeItemHeader itemHeader =
+        new SentryEnvelopeItemHeader(
+            SentryItemType.Statsd, () -> cachedItem.getBytes().length, null, null);
```

Comment:
do we need a contentType for this? we set it for everything else

Reply:
Good catch! Well it worked in my tests without, but I see python has a fallback to `application/octet-string`, I'll add it to my questions for later: https://github.com/getsentry/sentry-python/blob/e24508f94f1322bc95286d992e0ce3b9e5be3e7f/sentry_sdk/envelope.py#L195C16-L195C42